Configure

```diff
@@ -665,7 +666,6 @@ while (@argvcopy)
                         delete $disabled{"zlib"};
                         }
 		my $algo = $1;
-		delete $disabled{$algo};
```

Comment:
Why is this line being deleted?

Reply:
That's an oops...  I had some experimental code replacing it, but the experiment was cumbersome, so I reverted everything except this line.  Will fix.